tac/project-updates/annual-review-instructions.md

```diff
@@ -0,0 +1,28 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Annual Review Instructions
+Create a file in the `tac/project-updates` subdirectory for the current year and name the file `YYYY-annual-Project-Name.md` (e.g., `tac/project-updates/2024/2024-annual-lf-decentralized-trust-amazing.md`). Update the `mkdocs.yml` document in the root directory to include the file in the `nav` section under the appropriate year and quarter. If the year or quarter does not yet exist, please add it to match what you see in the `Project Updates` section.
+
+# What Should the Annual Review Contain
+Unlike the quarterly updates, the annual review process will be a "big picture" assessment of the project by reviewing the progress on goals from the prior year and goals for the next year. In addition, the annual review process will include a deep dive into the project health, including contributor and maintainer diversity and adoption metrics, and will include an assessment of the current project lifecycle state. The review will result in a set of recommendations for the project to improve and/or recommendation to move a project to another stage.
+
+The annual review should answer the following questions:
+
+- How did you progress against your goals for last year?
+- What deliverables/outputs did you have in the past year?
+- What are your goals for this year?
+- Do you need any help?
+- What is the Maintainer/Contributor diversity?
+
+# What Should the TAC Evaluate
+In general, the TAC will be looking at the health of the project overall and if it is in the correct lifecycle stage. The following is what the TAC will specifically look for:
```

Comment:
This is a nitpick, but I've noticed we use "lifecycle stage" in some places and "lifecycle state" elsewhere (Line 7 of this file, for example). I browsed the `project-lifecycle.md` file and found a mix of both words there too. Shall we standardize the terminology, selecting one word and using it everywhere?

Reply:
We should probably do this in a separate PR.

Reply:
Based on discussion at the TAC let's go with "lifecycle state"

Reply:
I'll submit a PR with the terminology updates.

Reply:
Fixed with https://github.com/LF-Decentralized-Trust/governance/pull/252